mkosi

```diff
@@ -2745,12 +2747,16 @@ def load_args():
         if args.secure_boot_certificate is None:
             die("UEFI SecureBoot enabled, but couldn't find certificate. (Consider placing it in mkosi.secure-boot.crt?)")
 
-    if args.verb in ("shell", "boot"):
+    if args.verb in ("shell", "boot", "qemu"):
```

Comment:
This seems redundant with the check below

Reply:
Hmm? not following?

I mean, the branch contains two checks, the tar check and the xz check. The latter is not redundant. The first one somewhat is, but thinking about the alternatives I like this way much better, and it outputs a friendlier, morespecific message anyway this way...

Reply:
Ah, indeed the compression is independent from the output format. Sorry for the noise.